### DISCUSSION TITLE
Game won't run

### DISCUSSION THREAD

**Discussion by tapinho0** [Aelto/W3_RandomEncounters_Tweaks#118] [Q&A] [answered]
Hi!

Trying to get the mod running. However, the game will not start due to the following error:

Error [mod_sharedutils_npcinteraction]local\sharedutils\npc_interaction.ws(156): 'global_event_handler' is not a member of '&handle:CPlayerInput'
Error [mod_sharedutils_npcinteraction]local\sharedutils\npc_interaction.ws(157): 'global_event_handler' is not a member of '&handle:CPlayerInput'
Error [mod_sharedutils_npcinteraction]local\sharedutils\npc_interaction.ws(161): 'global_event_handler' is not a member of '&handle:CPlayerInput'

Mods:

W3EE
W3EERedux
Gwent Redux
RER

![image](https://user-images.githubusercontent.com/108703328/177234298-f9208a94-62cb-43d9-805b-4cb034c830aa.png)

How to proceed?

**Comment by Aelto** [comment, marked as answer]
Hello! Refer to [the in-depth guide section 5 about merges](https://aelto.github.io/tw3-random-encounters-reworked/indepth-guide/#part-5), especiall the FHUD section. As stated in the EE section the overhaul comes with FHUD bundled in.

**Reply by tapinho0** [reply]
Many thanks!
